tutorial/Lab2/README.md

```diff
@@ -0,0 +1,151 @@
+
+Theory over, let's give it a go!
+
+1. Update the application
+
+    ```helm upgrade guestbook-demo ./guestbook --set redis.slaveEnabled=false,service.type=NodePort --namespace helm-demo```
+    
+    Helm upgrade takes an existing release and upgrades it according to the information you provide. You should see output similar to the following:
+    
+    ```console
+    Release "guestbook-demo" has been upgraded. Happy Helming!
+    LAST DEPLOYED: Mon Sep 24 10:36:18 2018
+    NAMESPACE: helm-demo
+    STATUS: DEPLOYED
+    
+    RESOURCES:
+    ==> v1/Service
+    NAME            AGE
+    guestbook-demo  1h
+    redis-master    1h
+    
+    ==> v1/Deployment
+    guestbook-demo  1h
+    redis-master    1h
+    
+    ==> v1/Pod(related)
+    
+    NAME                           READY  STATUS   RESTARTS  AGE
+    guestbook-demo-6c9cf8b9-dhqk9  1/1    Running  0         1h
+    guestbook-demo-6c9cf8b9-zddn2  1/1    Running  0         1h
+    redis-master-5d8b66464f-g7sh6  1/1    Running  0         1h
+    
+    
+    NOTES:
+    Get the application URL by running these commands:
+    export NODE_PORT=$(kubectl get --namespace helm-demo -o jsonpath="{.spec.ports[0].nodePort}" services guestbook-demo)
+    export NODE_IP=$(kubectl get nodes -o jsonpath={.items[*].status.addresses[?\(@.type==\"ExternalIP\"\)].address})
+    echo http://$NODE_IP:$NODE_PORT
+    ```
+    
+    The `upgrade` command upgrades the app to a specified version of a chart, removes the `redis-slave` resources and updates the app 
+    `service.type` to `NodePort`.
+        
+    To check the updates, you can run ```kubectl get all --namespace helm-demo```
+    
+    ```console
+    NAME                                READY     STATUS    RESTARTS   AGE
+    pod/guestbook-demo-6c9cf8b9-dhqk9   1/1       Running   0          1h
+    pod/guestbook-demo-6c9cf8b9-zddn2   1/1       Running   0          1h
+    pod/redis-master-5d8b66464f-g7sh6   1/1       Running   0          1h
+    
+    NAME                     TYPE        CLUSTER-IP      EXTERNAL-IP   PORT(S)          AGE
+    service/guestbook-demo   NodePort    172.21.43.244   <none>        3000:31202/TCP   1h
+    service/redis-master     ClusterIP   172.21.12.43    <none>        6379/TCP         1h
+    
+    NAME                             DESIRED   CURRENT   UP-TO-DATE   AVAILABLE   AGE
+    deployment.apps/guestbook-demo   2         2         2            2           1h
+    deployment.apps/redis-master     1         1         1            1           1h
+    
+    NAME                                      DESIRED   CURRENT   READY     AGE
+    replicaset.apps/guestbook-demo-6c9cf8b9   2         2         2         1h
+    replicaset.apps/redis-master-5d8b66464f   1         1         1         1h
+    ```
+    Note: The service type has changed (to `NodePort`) and a new port has been allocated (`31202` in this output case) to the guestbook 
+    service. All `redis-slave` resources have been removed.
```

Comment:
Move `service` to same line after `guestbook`.

Reply:
Done